### ISSUE TITLE
Resolve AppVeyor xfails

### ISSUE THREAD

**Issue by bbengfort** [DistrictDataLabs/yellowbrick#393]
In #386 we added an AppVeyor configuration but mostly resolved image comparison failures by marking them as `xfail` (thinking they were the product of different operating systems producing different types of images). 

In reality, there are a number of images that have some variability and have increased tolerances that actually do not support tests (see any tolerance of >=10) in the code. 

Once we get #379 working we can start to diagnose these issues in detail, and hopefully start resolving the `xfail` markers to ensure our tests run on all platforms.

**Comment by nickpowersys** [comment]
I analyzed the number of tests on Appveyor that with failing image comparisons. Just for analysis purposes, I modified these checks so that they did not lead to xfails (replacing `win32` with a nonsense replacement string). This allowed them to fail if the RMSE was greater than the default value of `tol`.

Based on the failing image comparisons, about 90% of the checks for `sys.platform == win32` are no longer needed, based on the updates to baseline images with x and y labels and legends removed in #823. 24 tests still exceeded the default `tol` with regular Python, and 31 tests exceeded it with Miniconda. Roughly 200 of the xfails are therefore not needed for Windows, assuming these regular Python and Miniconda test failures overlap closely. The tests with Appveyor RMSE>tol represent 2.7-3.5% of the total image comparison tests.
